src/main/java/com/netflix/imflibrary/st2067_2/CompositionImageEssenceDescriptorModel.java

```diff
@@ -384,10 +384,13 @@ public TransferCharacteristic getTransferCharacteristic() {
         Integer refPixelBitDepth = null;
         DOMNodeObjectModel subDescriptors = imageEssencedescriptorDOMNode.getDOMNode(regXMLLibDictionary.getSymbolNameFromURN(subdescriptorsUL));
         if (subDescriptors == null) {
-            imfErrorLogger.addError(IMFErrorLogger.IMFErrors.ErrorCodes.APPLICATION_COMPOSITION_ERROR,
-                    IMFErrorLogger.IMFErrors.ErrorLevels.NON_FATAL,
-                    String.format("EssenceDescriptor with ID %s is missing SubDescriptors", imageEssencedescriptorID.toString()));
-
+            Integer componentDepth = getComponentDepth();
```

Comment:
Section 6.1.2.4.1 of ST-2067-20:2016 indicates: 
"The Top-Level File Package of the Image Track File shall reference a JPEG 2000 Picture Sub Descriptor
SMPTE ST 422 as constrained by Table 13."
So it is an error in App2 and App2e (in the current published editions) to not have a subdescriptor. If this is relaxed, there should be a new namespace, etc.

Reply:
does it means I need to create also a CompositionImageEssenceDescriptorModel for the TSP 2121-1 ?